cmds/tui.c

```diff
@@ -1212,8 +1231,17 @@ static void win_display_graph(struct tui_window *win, void *node)
 		}
 
 		if (width < COLS) {
-			w = COLS - width;
-			printw("%-*.*s", w, w, curr->n.name);
+			if (curr->n.loc) {
+				printw("%s", curr->n.name);
+				width += strlen(curr->n.name);
+
+				asprintf(&str_loc, " [%s:%d]", curr->n.loc->file->name, curr->n.loc->line);
```

Comment:
Did you get crash when running uftrace tui without --srcline option?
that case makes crash due to my fault of last update. I'll fix it. thank you.
if you get crash with --srcline option, please let me know. thank you.

Reply:
It crashes regardless of having `--srcline` option. I found that `curr->n.loc` value is wrong for reasons.
```
(gdb) p curr->n.loc
$3 = (struct debug_location *) 0xffffffffffffffff
```

Reply:
I recommend you to record more complicated program rather than testing with `t-abc` only.

Reply:
if you can't share information of the program you use, could you please check it with latest update again ?
I tested with node, but it's not produced with --srcline option. 
the address you found is -1, so I think It could be related with code that is removed in latest update. 
If it's still produced, I'll try to look up programs for producing it. 
Thank you.